responsibleai/responsibleai/_interfaces.py

```diff
@@ -75,9 +75,36 @@ class CausalMetric:
     zstat: float
 
 
+class CausalPolicyGains:
+    recommended_policy_gains: float
+    treatment_gains: List[float]
+
+
+class CausalPolicyTreeLeaf:
+    leaf: bool
+    n_samples: int
+    treatment: str
+
+
+class CausalPolicyTreeInternal:
+    leaf: bool
+    feature: str
+    threshold: Union[float, str]  # TODO: Categorical features
+    left: Union['CausalPolicyTreeInternal', CausalPolicyTreeLeaf]
```

Comment:
nit: maybe make this string a constant

Reply:
We won't be using this anywhere else as it is a type hint for a recursive class definition. Normally I'd agree with you about constants because you never know when you'll want to reuse them! However, this is a rare case where reuse is highly unlikely.